addons/main/functions/fnc_3den_doNetworkConnection.sqf

```diff
@@ -0,0 +1,25 @@
+params ["_type", "_entity0", "_entity1"];
+
+// _entity0 = from object
+// _entity1 = to object
+
+//diag_log [_type, _entity0, _entity1];
+
+[_entity0, _entity1] spawn 
+{
+    params ["_entity0", "_entity1"];
+    waitUntil { !isNil "BIS_fnc_init" };
```

Comment:
Probably unnecessary if we wait for the variables to be set?

Reply:
Yes, you are right. I will remove that.